esti/system_test.go

```diff
@@ -150,21 +151,81 @@ func uploadFileRandomDataAndReport(ctx context.Context, repo, branch, objPath st
 }
 
 func uploadFileAndReport(ctx context.Context, repo, branch, objPath, objContent string, direct bool) (checksum string, err error) {
+	// Upload using direct access
 	if direct {
-		stats, err := helpers.ClientUploadDirect(ctx, client, repo, branch, objPath, nil, "", strings.NewReader(objContent))
+		stats, err := uploadContentDirect(ctx, client, repo, branch, objPath, nil, "", strings.NewReader(objContent))
```

Comment:
why you didn't remove the direct flag from the `uploadFileAndReport` and `uploadFileRandomDataAndReport` functions?

Reply:
There are specific GC tests that upload direct so I've moved the implementation and kept the test helper function.